components/ModalCard/index.jsx

```diff
@@ -0,0 +1,59 @@
+import React, {PropTypes} from 'react';
+import classnames from 'classnames';
+import {
+  modifierClassList, defaultReactProps, defaultReactPropsValues
+} from '../utils';
+
+const ModalCardFooter = ({children}) => {
+  return (
+    <footer className="modal-card-foot">
+      {children}
+    </footer>
+  );
+};
+
+ModalCardFooter.propTypes = {
+  children: PropTypes.arrayOf(PropTypes.element)
```

Comment:
This shouldn't be an array, as there may be a situation when a single element is passed, which would cause this to fail validation.